runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -319,6 +334,31 @@ public void init(@Nullable final Applet client) throws Exception
 
 			titleToolbar = new ClientTitleToolbar();
 			frame.add(container);
+
+			if (client != null)
+			{
+				client.addComponentListener(new ComponentAdapter()
```

Comment:
Why are you adding listeners to Client instead of `frame`? Just add listener to window and get size of client, because client can resize also when sidebar is opened/closed, and i am not sure if you want to save every time.

Reply:
Hmm... So I changed it to use the frame as the event listener, and a bug was introduced where the window changes position while dragging.
https://streamable.com/c26rf
(notice the right side of the frame)

This is how it looks like when using the client for the event listener:
https://streamable.com/h09il

I think part of the issue is that the client's size doesn't always update at the same time of the frame's size. So I think I'm gonna put it back to using the client as the event listener.

Reply:
From the records I see that even when using client as listener exactly same issue is present, just in a lot smaller scale, but it should not be present at all.